scripts/run.sh

```diff
@@ -62,44 +49,49 @@ function verify {
     ansible-playbook -i hosts verify.yml
     destroy
 }
-
+# adding user tags to all nodes in cluster?
```

Comment:
I think this would be advantageous. I deployed mine with tags and the worker nodes were left untagged. Was just a bit confused by this. 

I think I copied mine from another cluster playbook, but here's what I did in the `build` function for spark:

https://github.com/josephcardillo/marketplace-apache-spark-occ/blob/4da1709bc2b759e2d18f0ce609366e3a1093794a/scripts/run.sh#L41C1-L42C1
https://github.com/josephcardillo/marketplace-apache-spark-occ/blob/4da1709bc2b759e2d18f0ce609366e3a1093794a/scripts/run.sh#L60